examples/bri-3/src/bri/workgroup/workgroups/api/workgroups.controller.spec.ts

```diff
@@ -43,17 +43,16 @@ describe('WorkgroupsController', () => {
       'name',
       'desc',
       BpiSubjectType.External,
-      'publicKey',
+      '0x9a36744bCD7fb5b41550e4719FFeA8B1B8c6A7bE',
```

Comment:
see above

Reply:
Although this file has been altered these hardcoded values are used to instantiate entities within mock storage. Each controller spec tests uses hardcoded entities for mocks of storage.

Reply:
issue created